jupyter_core/command.py

```diff
@@ -122,12 +122,12 @@ def _jupyter_abspath(subcommand):
     abs_path = which(jupyter_subcommand, path=search_path)
     if abs_path is None:
```

Comment:
Should we filter out subcommands == 'help' as a special case to avoid `jupyter help` printing `Jupyter command 'jupyter-help' not found.`?  This would essentially make `jupyter help` == `jupyter --help`.  I can see both arguments, but this seems a bit more friendly.